blobfuse/blobfuse.cpp

```diff
@@ -171,7 +169,8 @@ int read_config(std::string configFile)
 
 void *azs_init(struct fuse_conn_info * conn)
 {
-    azure_blob_client_wrapper = std::make_shared<blob_client_wrapper>(blob_client_wrapper::blob_client_wrapper_init(str_options.accountName, str_options.accountKey, 20, str_options.use_https, str_options.blobEndpoint));
+    azure_blob_client_wrapper = std::make_shared<blob_client_wrapper>(blob_client_wrapper::blob_client_wrapper_init(str_options.accountName, str_options.accountKey, str_options.sasToken, 20/*concurrency*/, str_options.use_https, 
+														    str_options.blobEndpoint));
```

Comment:
Formatting looks weird here, but maybe that's just Github?

Reply:
I think this and some of the other formatting comments are GitHub. In my editor, str_options.accountName and str_options.blobEndpoint align

Reply:
Please configure your editor to use spaces instead of tabs